### PR TITLE
Update taxonomy.simple.yaml

### PR DESCRIPTION
**Pull request by morchickit**
Changed based on comments here - 
https://github.com/kolzchut/srm/issues/714#issuecomment-2022434490

### CODE REVIEW

File: taxonomy.simple.yaml [hunk 1]
```diff
@@ -36,14 +36,18 @@
...
   - Education:
     - Afterschool Care
-    - Respite Activities:
-      - Further Education:
-        - Engineering
-        - Yeshiva and Kollel
-      - Academia:
+    - Nursery
+    - Respite Activities
+    - Further Education
```
Comment (akariv) [inline taxonomy.simple.yaml:41]: missing :

File: taxonomy.simple.yaml [hunk 2]
```diff
@@ -36,14 +36,18 @@
...
+      - Engineering
+      - Yeshiva and Kollel
+    - Academia
```
Comment (akariv) [inline taxonomy.simple.yaml:44]: missing :

File: taxonomy.simple.yaml [hunk 3]
```diff
@@ -97,8 +101,7 @@
       - Substance Abuse Counseling
       - Support groups
     - Dental Care
```
Comment (akariv) [inline taxonomy.simple.yaml:103]: missing :

File: taxonomy.simple.yaml [hunk 4]
```diff
@@ -110,10 +113,12 @@
         - Individual Counseling
       - Medications for Mental Health
       - Mental Health Evaluation
+      - Family advice centre
       - Hospitalization:
         - Forced Hospitalization
         - Voluntary Hospitalization
         - Hospitalization Alternative
+          - Balanching House
```
Comment (akariv) [inline taxonomy.simple.yaml:121]: "Balancing" (typo)

File: taxonomy.simple.yaml [hunk 5]
```diff
@@ -110,10 +113,12 @@
         - Individual Counseling
       - Medications for Mental Health
       - Mental Health Evaluation
+      - Family advice centre
```
Comment (akariv) [inline taxonomy.simple.yaml:116]: 1. Capitalize
2. use "Center", not "Centre"

File: taxonomy.simple.yaml [hunk 6]
```diff
@@ -284,21 +294,29 @@
...
     - Ex-Offenders youth
     - Prisoners
     - Ex prisoners 
     - Sexual Perpetrators
     - Violent tendencies
+ 
   - Deprivation:
-    - At risk youth
+    - At risk:
+     - At risk children
```
Comment (akariv) [inline taxonomy.simple.yaml:312]: indent one space here

Comment (akariv) [reply taxonomy.simple.yaml:312]: Also capitalize

File: taxonomy.simple.yaml [hunk 7]
```diff
@@ -284,21 +294,29 @@
...
+     - At risk teens
+     - At risk girls and teens
+     - At risk youtth
```
Comment (akariv) [inline taxonomy.simple.yaml:315]: typo

File: taxonomy.simple.yaml [hunk 8]
```diff
@@ -284,21 +294,29 @@
...
     - Ex prisoners 
     - Sexual Perpetrators
     - Violent tendencies
+ 
   - Deprivation:
-    - At risk youth
+    - At risk:
+     - At risk children
+     - At risk teens
+     - At risk girls and teens
```
Comment (akariv) [inline taxonomy.simple.yaml:314]: at risk teen girls?

File: taxonomy.simple.yaml [hunk 9]
```diff
@@ -468,18 +500,15 @@
...
-      - Youth Villages
+    - High School
   - Health:
     - Clinic: 
       - Mental Health Clinic
       - GP Clinic
       - Well Baby Clinic
     - Hospital 
   - Government Service Point
+  - Welfare bureaus
```
Comment (akariv) [inline taxonomy.simple.yaml:514]: Capitalize

File: taxonomy.simple.yaml (hunk 1 above)
Comment (morchickit) [reply taxonomy.simple.yaml:41]: No missing, the : was a mistake from previous pull request! :-0

File: taxonomy.simple.yaml [hunk 10]
```diff
@@ -110,10 +113,12 @@
         - Individual Counseling
       - Medications for Mental Health
       - Mental Health Evaluation
+      - Family advice center
       - Hospitalization:
         - Forced Hospitalization
         - Voluntary Hospitalization
         - Hospitalization Alternative
```
Comment (akariv) [inline taxonomy.simple.yaml:120]: missing ':'

File: taxonomy.simple.yaml [hunk 11]
```diff
@@ -36,14 +36,18 @@
...
-        - Engineering
-        - Yeshiva and Kollel
-      - Academia:
+    - Nursery
+    - Respite Activities
+    - Further Education:
+      - Engineering
+      - Yeshiva and Kollel
+    - Academia:
         - College Readiness
```
Comment (akariv) [inline taxonomy.simple.yaml:45]: De-indent by 2 spaces

File: taxonomy.simple.yaml [hunk 12]
```diff
@@ -110,10 +113,12 @@
         - Individual Counseling
       - Medications for Mental Health
       - Mental Health Evaluation
+      - Family advice center
```
Comment (akariv) [inline taxonomy.simple.yaml:116]: Capitalize

File: taxonomy.simple.yaml [hunk 13]
```diff
@@ -284,21 +294,29 @@
...
     - Ex-Offenders youth
     - Prisoners
     - Ex prisoners 
     - Sexual Perpetrators
     - Violent tendencies
+ 
   - Deprivation:
-    - At risk youth
+    - At risk:
+      - At risk children
```
Comment (akariv) [inline taxonomy.simple.yaml:312]: Capitalize + fix typo

File: taxonomy.simple.yaml [hunk 14]
```diff
@@ -468,18 +500,15 @@
...
-      - Youth Villages
+    - High School
   - Health:
     - Clinic: 
       - Mental Health Clinic
       - GP Clinic
       - Well Baby Clinic
     - Hospital 
   - Government Service Point
+  - Welfare Bureaus
```
Comment (akariv) [inline taxonomy.simple.yaml:514]: Change to singular instead of plural

**Pull request merged**